src/Microsoft.Identity.Abstractions/TokenAcquisition/AcquireTokenOptions.cs

```diff
@@ -90,6 +92,39 @@ public AcquireTokenOptions(AcquireTokenOptions other)
         /// </summary>
         public string? PopClaim { get; set; }
 
+        /// <summary>
+        /// When <see cref="ManagedIdentity"/> is set, the application uses a managed identity instead of client credentials to
+        /// acquire an app token.<br/><br/>
```

Comment:
remove `<br/><br/>`

Reply:
The break tags are to make the explanation easier to read in the IDE hover-over text, is there a reason we should remove them?

With breaks:
![IdeTipWithBreaks](https://github.com/AzureAD/microsoft-identity-abstractions-for-dotnet/assets/103777376/b9694ae0-e618-419d-a17b-a7a99fe5d906)

Without breaks:
![IdeTipWithoutBreaks](https://github.com/AzureAD/microsoft-identity-abstractions-for-dotnet/assets/103777376/4b129659-28ee-4019-9c87-319f446cdd90)

Reply:
I didn't know about that.
Thanks for educating us, @JoshLozensky

Reply:
synced offline, for consistency and how this impacts other docs, will remove.